Classes/Plugin.php

```diff
@@ -34,8 +34,14 @@ public function on($eventKey, $data = null) {
 			// limit words function -- very rough limit due to HTML input
 			$limit_words = new \Twig_SimpleFilter('limit_words', function ($string) {
 				$words = str_word_count($string, 2);
+				$nbwords = str_word_count($string, 0);
```

Comment:
Can you change this to `$nbwords = count($words);`?

Just a small performance thing. While `str_word_count()` isn't exactly slow it is much more costly then a simple array count.